app/assets/stylesheets/participants.scss

```diff
@@ -0,0 +1,5 @@
+table.participants {
+  input[type=text], input[type=number], select {
+    min-width: 5rem;
```

Comment:
had to look up what `rem` is, thought it was a typo. Interesting

Reply:
Interestingly, I'm starting to doubt myself here. I think `em` might be a better choice,